docs/contributing/style-guide.rst

```diff
@@ -63,21 +63,5 @@ Example Docstring
 
 Encoding
```

Comment:
While you're editing this document, above on line 29, the link to Google's Python Style Guide is broken and yields a 404 Error.  Removing showone=Comments#Comments fixes the link.